### PR TITLE
Add config and setup for matomo

### PR DESCRIPTION
**Pull request by HarryBosworth**
- Adds two env vars PIWIK_URL and SITE_ID for matomo
- Adds revelevant scripts to the templates
- Passes cofig throught to the template
- Sets the view name in a hidden dev for matomo 

Related docker-chs-development PR: https://github.com/companieshouse/docker-chs-development/pull/1167
Related ecs-service-configs PR: https://github.com/companieshouse/ecs-service-configs-dev/pull/465

Resolves: [AOAF-404](https://companieshouse.atlassian.net/browse/AOAF-404)

[AOAF-404]: https://companieshouse.atlassian.net/browse/AOAF-404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -42,7 +42,9 @@ njk.addGlobal("ExternalUrls", ExternalUrls);
 njk.addGlobal("Urls", Urls);
 njk.addGlobal("servicePathPrefix", servicePathPrefix);
 njk.addGlobal("PrefixedUrls", PrefixedUrls);
-
+njk.addGlobal("SERVICE_NAME", 'presnter-account-web');
```
Comment (amartin7663) [inline src/app.ts:45]: Type in the service name

File: src/routers/handlers/confirmation/index.ts [hunk 2]
```diff
@@ -8,14 +8,16 @@ interface ConfirmationViewData extends BaseViewData {
 
 export class ConfirmationHandler extends GenericHandler<ConfirmationViewData> {
     public static templatePath = "router_views/confirmation/confirmation";
+    private static viewName = 'confirmation';
```
Comment (amartin7663) [inline src/routers/handlers/confirmation/index.ts:11]: This feels a bit overkill could we just have ``` viewName: 'confirmation' ``` in the return

File: src/views/partials/__meta_header.njk [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <link href="{{ cdnHost }}/stylesheets/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.css" rel="stylesheet"/>
     <link href="{{ cdnHost }}/stylesheets/services/presenter-account/application.css" rel="stylesheet"/>
 
-    <link href="{{ cdnUrlCss }}/app.min.css" media="all" rel="stylesheet" type="text/css" />
+    {# <link href="{{ cdnUrlCss }}/app.min.css" media="all" rel="stylesheet" type="text/css" /> #}
```
Comment (amartin7663) [inline src/views/partials/__meta_header.njk:10]: This has been failing for ages, let's just delete it if we don't need it

File: src/views/partials/signout-bar.njk [hunk 4]
```diff
@@ -4,5 +4,4 @@
         <span class="content-email" id="signed-in-user">{{email}}</span>
         <a class="govuk-link" href="{{ExternalUrls.SIGNOUT}}" data-event-id="sign-out-button-selected" id="user-signout">Sign Out</a>
     </div>
-<script src="{{ cdnHost }}/javascripts/app/session-timeout.js"></script>
```
Comment (amartin7663) [inline src/views/partials/signout-bar.njk:7]: Don't we need this though?

Comment (HarryBosworth) [reply src/views/partials/signout-bar.njk:7]: [I moved it into the footer](https://github.com/companieshouse/presenter-account-web/blob/797b53f323ec8b4dab9a0026997641d1ace6524c/src/views/partials/__footer.njk#L48) with the rest of the scripts so they only run after the HTML is loaded. If preferred, I could keep it here with the signout bar and add the `defer` attribute to achieve the same effect, but keep it with the signout bar HTML.

**Review by amartin7663 — APPROVED** [review]

**Pull request merged**